### PR TITLE
Add fpgun_ar_ammo_blend command

### PR DESCRIPTION
**Pull request by natarii**
I'm not entirely happy with this, but there is some demand (issue #119) so I figured I'd put it together quick and send a PR.

### CODE REVIEW

**Comment by natarii** [comment]
After some discussion in Discord this might need to be re-thunk :)

This would be hacky, but it might be a good overall compromise to use the stock behavior if the digits vbm cksum matches stock files, and to patch it to allow any color if the vbm has been modified. Always allowing any color wouldn't work since the stock vbm is actually yellow... but by changing the behavior based on the checksum, the default behavior is preserved without any changes to game files, but new vbms can still override it.

Basically I'm thinking it makes more sense to be able to bundle a new vbm with an fpgun skin mod, than to tell people to use a console command. This also would make it a little more user-friendly when running mods.

**Comment by rafalh** [comment]
I don't like idea of using a console command for it. Property in weapons.tbl would be better so PS2 conversion mod could easily overwrite it. But it would create another issue, inability to customize it and still play normal multi-player servers... Perhaps your idea with checking if vbm was overridden makes more sense. The only issue with that is that PS2 mod must include that file with some dummy modification even if default file is fine for it (yellow). But this file is only 20KB so it's not a big deal.

**Comment by rafalh** [comment]
Closing it for now because AFAIK Goober has a change to put it into tbl file. Feel free to reopen if you want to work on it
